components/basic-transformers/src/main/java/org/datacleaner/beans/codec/XmlEncoderTransformer.java

```diff
@@ -19,24 +19,18 @@
  */
 package org.datacleaner.beans.codec;
 
-import javax.inject.Named;
-
-import org.datacleaner.api.Categorized;
-import org.datacleaner.api.Configured;
-import org.datacleaner.api.Description;
-import org.datacleaner.api.InputColumn;
-import org.datacleaner.api.InputRow;
-import org.datacleaner.api.OutputColumns;
-import org.datacleaner.api.Transformer;
+import com.google.common.escape.Escaper;
+import com.google.common.xml.XmlEscapers;
+import org.datacleaner.api.*;
 import org.datacleaner.components.categories.StringManipulationCategory;
 import org.datacleaner.data.MockInputColumn;
 
-import com.google.common.escape.Escaper;
-import com.google.common.xml.XmlEscapers;
+import javax.inject.Named;
```

Comment:
There seem to be some code style differences here. I guess it is the difference between Eclipse and IntelliJ: Eclipse puts _all_ top-level packages in its own group, IntelliJ only puts a few (java and javax comes to mind).

I had to add top-level packages found ( .com, .org, etc) to my IntelliJ code style configuration, which is a bit of a pain, especially since it is an ongoing process.

![image](https://cloud.githubusercontent.com/assets/1282832/9515356/411f7e06-4ca1-11e5-889a-96d54b6b0fa0.png)